licenses/admin.py

```diff
@@ -1,30 +1,63 @@
 from django.contrib import admin
 
-from licenses.models import License, LicenseClass, Language, Jurisdiction
+from licenses.models import LegalCode, License
 
 
-@admin.register(License)
-class LicenseAdmin(admin.ModelAdmin):
+@admin.register(LegalCode)
+class LegalCodeAdmin(admin.ModelAdmin):
+    fields = [
+        "url",
+        "language_code",
+    ]
+    list_display = [
+        "url",
+        "language_code",
+    ]
     list_filter = [
-        "license_code",
-        "version",
-        "creator__url",
-        "license_class__url",
-        "jurisdiction__code",
-        "jurisdiction__default_language__code",
+        "language_code",
     ]
 
 
-@admin.register(LicenseClass)
-class LicenseClassAdmin(admin.ModelAdmin):
-    pass
-
+class LegalCodeInline(admin.TabularInline):
+    model = LegalCode
 
-@admin.register(Language)
-class LanguageAdmin(admin.ModelAdmin):
-    pass
 
-
-@admin.register(Jurisdiction)
-class JurisdictionAdmin(admin.ModelAdmin):
-    list_display = ["code", "default_language"]
+@admin.register(License)
+class LicenseAdmin(admin.ModelAdmin):
+    fields = [
+        "about",
```

Comment:
https://cc-vocabulary.netlify.app/?path=/docs/tokens-icons--figma

can we turn this list into a tuple and store the associated icon class where applicable. For example `(requires_share_alike, cc-sa)`. I think it will be helpful for pulling the relevant icons when displaying the deed and license.